code/datums/components/construction.dm

```diff
@@ -87,8 +87,9 @@
 				. = user.transferItemToLoc(I, parent)
 
 			// Using stacks
-			else if(istype(I, /obj/item/stack))
-				. = I.use_tool(parent, user, 0, volume=50, amount=current_step["amount"])
+			else 
+				if(istype(I, /obj/item/stack))
+					. = I.use_tool(parent, user, 0, volume=50, amount=current_step["amount"])
```

Comment:
else if isn't really valid in a switch statement, see also:
https://github.com/tgstation/tgstation/pull/67036/files#r873321614

This may actually affect us- This is some general tg construction code, I'm unsure if we utilize this.